apps/microtvm/ethosu/run_demo.sh

```diff
@@ -148,6 +190,7 @@ python3 -m tvm.driver.tvmc compile --target=ethos-u,cmsis-nn,c \
     --target-ethos-u-accelerator_config=ethos-u55-256 \
     --target-cmsis-nn-mcpu=cortex-m55 \
     --target-c-mcpu=cortex-m55 \
+    --target-ethos-u-disable_copying_constants=1 \
```

Comment:
Maybe add information in the README that we use mode when weights should be placed in the SRAM.

Reply:
This is a demo and I'm not sure we need to go deep in describing specific parameters, because we already have pre-defined memory layout in *.ld files. Changing of the `--target-ethos-u-disable_copying_constants` parameter requires clear understanding Alif's memory and modification of LD scripts. So IMHO this topic is out of scope of the demo, but it could be a part of the another big document describing TVM on EthosU.